index.d.ts

```diff
@@ -210,9 +211,11 @@ declare module "replicate" {
           deployment_name: string,
           options: {
             input: object;
+            /** @deprecated */
             stream?: boolean;
             webhook?: string;
             webhook_events_filter?: WebhookEventType[];
+            block?: boolean;
```

Comment:
I think I preferred the version we had that exposed `wait` here. Where wait was one of:

```ts
wait?: { mode: "block", timeout?: number } | { mode: "poll", interval?: number }
```

This way we have only one param consistently, and it defaults to `{mode: "block"}`.